pallets/liquid-staking-v2/src/lib.rs

```diff
@@ -216,8 +204,57 @@ mod pallet {
         }
     }
 
+    #[pallet::hooks]
+    impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+        /// Try to pay off over the `UnstakeQueue` while blockchain is on idle.
+        ///
+        /// It breaks when:
+        ///     - Pallet's balance is insufficiant.
+        ///     - Queue is empty.
+        ///     - `remaining_weight` is less than one pop_queue needed.
+        fn on_idle(_n: BlockNumberFor<T>, remaining_weight: Weight) -> Weight {
+            let mut remaining_weight = remaining_weight;
+            let base_weight = T::WeightInfo::pop_queue();
+            loop {
+                // Check weight is enough
+                if remaining_weight < base_weight {
+                    break;
+                }
+
+                if Self::unstake_queue().is_empty() {
+                    break;
+                }
+
+                let (who, amount) = Self::unstake_queue()
+                    .first()
+                    .expect("Has been checked before.")
+                    .clone();
+
+                if T::Currency::transfer(
```

Comment:
we should define a variable here instead of writing it all together

Reply:
I think it's already simple and clear enough. I'll add more comment if you warried.